app/views/shared/_tag_manager.html.erb

```diff
@@ -1,4 +1,11 @@
 <% if ENV.include?("GOOGLE_TAG_MANAGER_ID") %>
+
+<script>
+  window.dataLayer = window.dataLayer || [
+    { "scrapbookId": "<%= current_scrapbook.id %>" }
+  ]
+</script>
```

Comment:
Think we might be missing something if this is already in the URL but instead we are removing it and the re adding it ..